Build failure: python3.11-dnspython-2.6.1

## Steps To Reproduce

```bash
git clone https://github.com/NixOS/nixpkgs.git
git switch nixos-24.05
nix-build -A python311Packages.dns
```


## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log</summary>

```
﻿this derivation will be built:
  /nix/store/ar2ryb4ighb0x0wlzkq0yb5947xa8slz-python3.11-dnspython-2.6.1.drv
building '/nix/store/ar2ryb4ighb0x0wlzkq0yb5947xa8slz-python3.11-dnspython-2.6.1.drv'...
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
Running phase: unpackPhase
unpacking source archive /nix/store/5h226hd26yy7p1l3ypi29x2bmgfabfnx-dnspython-2.6.1.tar.gz
source root is dnspython-2.6.1
setting SOURCE_DATE_EPOCH to timestamp 1580601600 of file dnspython-2.6.1/util/generate-rdatatype-doc.py
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
* Getting build dependencies for wheel...
* Building wheel...
Successfully built dnspython-2.6.1-py3-none-any.whl
Finished creating a wheel...
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for dnspython-2.6.1-py3-none-any.whl
Finished executing pythonRuntimeDepsCheck
Running phase: installPhase
Executing pypaInstallPhase
Successfully installed dnspython-2.6.1-py3-none-any.whl
Finished executing pypaInstallPhase
Running phase: pythonOutputDistPhase
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
Running phase: fixupPhase
checking for references to /private/tmp/nix-build-python3.11-dnspython-2.6.1.drv-0/ in /nix/store/qh1ql2bp14yg0wyjcf16sgbc0l501has-python3.11-dnspython-2.6.1...
patching script interpreter paths in /nix/store/qh1ql2bp14yg0wyjcf16sgbc0l501has-python3.11-dnspython-2.6.1
stripping (with command strip and flags -S) in  /nix/store/qh1ql2bp14yg0wyjcf16sgbc0l501has-python3.11-dnspython-2.6.1/lib
checking for references to /private/tmp/nix-build-python3.11-dnspython-2.6.1.drv-0/ in /nix/store/yzci558szqxjrl1qy7gppcpfpwn7g7s7-python3.11-dnspython-2.6.1-dist...
patching script interpreter paths in /nix/store/yzci558szqxjrl1qy7gppcpfpwn7g7s7-python3.11-dnspython-2.6.1-dist
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
Running phase: pythonRemoveBinBytecodePhase
Running phase: pythonImportsCheckPhase
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: dns
Running phase: pytestCheckPhase
Executing pytestCheckPhase
============================= test session starts ==============================
platform darwin -- Python 3.11.10, pytest-8.1.1, pluggy-1.4.0
rootdir: /private/tmp/nix-build-python3.11-dnspython-2.6.1.drv-0/dnspython-2.6.1
configfile: pyproject.toml
collected 1345 items / 1 deselected / 1344 selected                            

tests/test_address.py ....                                               [  0%]
tests/test_async.py ..........ss..............s......................    [  3%]
tests/test_bugs.py .......                                               [  4%]
tests/test_constants.py .....                                            [  4%]
tests/test_ddr.py ..                                                     [  4%]
tests/test_dnssec.py ................................................... [  8%]
..............                                                           [  9%]
tests/test_dnssecalgs.py .........                                       [ 10%]
tests/test_doh.py ssssss                                                 [ 10%]
tests/test_doq.py ss                                                     [ 11%]
tests/test_edns.py ...............                                       [ 12%]
tests/test_entropy.py .....                                              [ 12%]
tests/test_exceptions.py .....                                           [ 12%]
tests/test_features.py ....                                              [ 13%]
tests/test_flags.py ............                                         [ 14%]
tests/test_generate.py .................                                 [ 15%]
tests/test_grange.py ..........                                          [ 16%]
tests/test_immutable.py ......                                           [ 16%]
tests/test_message.py .................s................................ [ 20%]
........................                                                 [ 22%]
tests/test_name.py ............s.............s.......................... [ 26%]
..........................ss.........s.......................s.......... [ 31%]
.........................sss......................                       [ 35%]
tests/test_namedict.py ......................                            [ 36%]
tests/test_nsec3.py ..                                                   [ 36%]
tests/test_nsec3_hash.py ...                                             [ 37%]
tests/test_ntoaaton.py ................................................  [ 40%]
tests/test_processing_order.py ..........                                [ 41%]
tests/test_query.py .......................ssssssssssss................  [ 45%]
tests/test_rdata.py .................................................... [ 49%]
...........s..........................                                   [ 51%]
tests/test_rdataset.py ..s...............                                [ 53%]
tests/test_rdtypeandclass.py ..........................                  [ 55%]
tests/test_rdtypeanydnskey.py ..                                         [ 55%]
tests/test_rdtypeanyeui.py ........................                      [ 57%]
tests/test_rdtypeanyloc.py ....                                          [ 57%]
tests/test_rdtypeanytkey.py ....                                         [ 57%]
tests/test_renderer.py .....                                             [ 58%]
tests/test_resolution.py ...............................                 [ 60%]
tests/test_resolver.py ..............................s.................. [ 64%]
..s....................s........................sssssss                  [ 68%]
tests/test_resolver_override.py F.............                           [ 69%]
tests/test_rrset.py .s....................                               [ 70%]
tests/test_rrset_reader.py ............                                  [ 71%]
tests/test_serial.py ...........                                         [ 72%]
tests/test_set.py ...................................................    [ 76%]
tests/test_svcb.py ................                                      [ 77%]
tests/test_tokenizer.py ................................................ [ 81%]
                                                                         [ 81%]
tests/test_transaction.py .............................................. [ 84%]
.                                                                        [ 84%]
tests/test_tsig.py ....................                                  [ 86%]
tests/test_tsigkeyring.py .......                                        [ 86%]
tests/test_ttl.py .......                                                [ 87%]
tests/test_update.py .......................                             [ 88%]
tests/test_wire.py .......                                               [ 89%]
tests/test_xfr.py ...........................ssss                        [ 91%]
tests/test_zone.py ........s............................................ [ 95%]
................................................                         [ 99%]
tests/test_zonedigest.py ...........                                     [100%]

=================================== FAILURES ===================================
____________ OverrideSystemResolverTestCase.test_basic_getaddrinfo _____________

self = <tests.test_resolver_override.OverrideSystemResolverTestCase testMethod=test_basic_getaddrinfo>

    @unittest.skipIf(
        sys.platform == "win32", "avoid windows original getaddrinfo issues"
    )
    def test_basic_getaddrinfo(self):
        self.assertTrue(
            self.equivalent("dns.google", 53, socket.AF_INET, socket.SOCK_DGRAM)
        )
>       self.assertTrue(
            self.equivalent("dns.google", 53, socket.AF_INET6, socket.SOCK_DGRAM)
        )
E       AssertionError: False is not true

tests/test_resolver_override.py:104: AssertionError
----------------------------- Captured stdout call -----------------------------
NOT EQUIVALENT
[(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8844', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8888', 53, 0, 0))]
[(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.8.8', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.4.4', 53, 0, 0))]
=========================== short test summary info ============================
FAILED tests/test_resolver_override.py::OverrideSystemResolverTestCase::test_basic_getaddrinfo - AssertionError: False is not true
=========== 1 failed, 1292 passed, 51 skipped, 1 deselected in 6.48s ===========
error: builder for '/nix/store/ar2ryb4ighb0x0wlzkq0yb5947xa8slz-python3.11-dnspython-2.6.1.drv' failed with exit code 1;
       last 10 log lines:
       > E       AssertionError: False is not true
       >
       > tests/test_resolver_override.py:104: AssertionError
       > ----------------------------- Captured stdout call -----------------------------
       > NOT EQUIVALENT
       > [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8844', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8888', 53, 0, 0))]
       > [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.8.8', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.4.4', 53, 0, 0))]
       > =========================== short test summary info ============================
       > FAILED tests/test_resolver_override.py::OverrideSystemResolverTestCase::test_basic_getaddrinfo - AssertionError: False is not true
       > =========== 1 failed, 1292 passed, 51 skipped, 1 deselected in 6.48s ===========
       For full logs, run 'nix log /nix/store/ar2ryb4ighb0x0wlzkq0yb5947xa8slz-python3.11-dnspython-2.6.1.drv'.
```

</details>

## Additional context

<!-- Add any other context about the problem here. -->
Noticed the error when trying to update my nix darwin system. It's causing various packages to fail:

<details>

<summary>darwin-rebuild switch --flake /path/to/flake </summary>

```
error: builder for '/nix/store/ar2ryb4ighb0x0wlzkq0yb5947xa8slz-python3.11-dnspython-2.6.1.drv' failed with exit code 1;
       last 10 log lines:
       > E       AssertionError: False is not true
       >
       > tests/test_resolver_override.py:104: AssertionError
       > ----------------------------- Captured stdout call -----------------------------
       > NOT EQUIVALENT
       > [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8844', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8888', 53, 0, 0))]
       > [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.4.4', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.8.8', 53, 0, 0))]
       > =========================== short test summary info ============================
       > FAILED tests/test_resolver_override.py::OverrideSystemResolverTestCase::test_basic_getaddrinfo - AssertionError: False is not true
       > ========== 1 failed, 1292 passed, 51 skipped, 1 deselected in 18.69s ===========
       For full logs, run 'nix log /nix/store/ar2ryb4ighb0x0wlzkq0yb5947xa8slz-python3.11-dnspython-2.6.1.drv'.
error: 1 dependencies of derivation '/nix/store/p0mz7pdl4phzw4mh37prapf6dgd2sjxx-python3.11-email-validator-2.1.1.drv' failed to build
error: 1 dependencies of derivation '/nix/store/bis1vfh2iv3q25f8kdnxhbagp9ldd2la-python3.11-eventlet-0.35.2.drv' failed to build
error: 1 dependencies of derivation '/nix/store/8k25fiybk5c5qlyv8cjwbcrp750r71dj-python3.11-pymongo-4.6.3.drv' failed to build
error: 1 dependencies of derivation '/nix/store/ilqipg2651gsrnhrsmd59mf9q7g45ilg-python3.11-gunicorn-21.2.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/xv0rg4jfp5cam7cgz4m650imrxl07qkg-python3.11-motor-3.4.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/gxqqqd86pfflf3yx2zfnaflsqcm6dl8r-python3.11-pydantic-2.6.3.drv' failed to build
error: 1 dependencies of derivation '/nix/store/xr0jvhj6i64kycldhw8sy142fbxk121n-python3.11-aiohttp-3.9.5.drv' failed to build
error: 1 dependencies of derivation '/nix/store/0z3cxvpxa8yicvjkd7qdcicm0x748p00-python3.11-cattrs-23.2.3.drv' failed to build
error: 1 dependencies of derivation '/nix/store/vycpdbwjwb29hc165d4jzfkvnidl8yif-python3.11-inflect-7.2.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/202l0i9s7wy6xdr6k3ai99q59msagjji-python3.11-black-24.4.0.drv' failed to build
error (ignored): error: cannot unlink '/private/tmp/nix-build-yaml-language-server-1.14.0.drv-15': Directory not empty
error: 1 dependencies of derivation '/nix/store/866ni3f677bydfqpsl0hspmbga9p17ja-python3.11-elastic-transport-8.13.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/8q4yvkqf272jzf2958vr0jynvaz3js6x-python3.11-lsprotocol-2023.0.1.drv' failed to build
error: 1 dependencies of derivation '/nix/store/h03si468mlmc2f569jp972vgh3ijlix1-neovim-0.9.5.drv' failed to build
error (ignored): error: cannot unlink '/private/tmp/nix-build-ghostscript-10.04.0.drv-20/ghostscript-10.04.0': Directory not empty
error: 1 dependencies of derivation '/nix/store/i18qw6xsygdg89aqhavn1wgsh8kfwy1b-python3.11-pytest-benchmark-4.0.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/l8fwjac2i448k4v1q9bfs02bk4nvkqw0-home-manager-applications.drv' failed to build
error: 1 dependencies of derivation '/nix/store/vapdbnqkd3cdvgh4cyym930hbbwvwd4s-home-manager-fonts.drv' failed to build
error: 1 dependencies of derivation '/nix/store/y432n6b5av4rb061j0ghy004qwk73lzi-home-manager-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/fazl09fm2959z127yxj5hd4ml4nx0gls-home-manager-generation.drv' failed to build
error: 1 dependencies of derivation '/nix/store/kr9hgwp2sn2i9s8lgzdl55zwsl70srf6-user-environment.drv' failed to build
error: 1 dependencies of derivation '/nix/store/bfjhpdn62d3g7k6gn92lva90ramsmv7r-activation-harryp.drv' failed to build
error: 1 dependencies of derivation '/nix/store/l1wv8kkhh04gmi4dsqhibmna8an17n4p-darwin-system-24.05.20241116.e8c38b7+darwin4.34588d5.drv' failed to build
```
</details>

Here is my config if it helps: https://github.com/HPRIOR/nix-config. Very new to nix so wouldn't be suprised if the error is caused by my setup.


## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->
 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.1.0, macOS 15.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.18.8`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/store/dxdcvjnvz3b91gvsrhpb7gp156nnj8bf-source`

## Notify maintainers
@gador 

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

Getting the same error with: `nix build github:nixos/nixpkgs/release-24.05/#python311Packages.dnspython`

<details>
error: builder for '/nix/store/ar2ryb4ighb0x0wlzkq0yb5947xa8slz-python3.11-dnspython-2.6.1.drv' failed with exit code 1;
       last 10 log lines:
       > E       AssertionError: False is not true
       >
       > tests/test_resolver_override.py:104: AssertionError
       > ----------------------------- Captured stdout call -----------------------------
       > NOT EQUIVALENT
       > [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8844', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8888', 53, 0, 0))]
       > [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.4.4', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.8.8', 53, 0, 0))]
       > =========================== short test summary info ============================
       > FAILED tests/test_resolver_override.py::OverrideSystemResolverTestCase::test_basic_getaddrinfo - AssertionError: False is not true
       > =========== 1 failed, 1292 passed, 51 skipped, 1 deselected in 6.42s ===========
       For full logs, run 'nix log /nix/store/ar2ryb4ighb0x0wlzkq0yb5947xa8slz-python3.11-dnspython-2.6.1.drv'.
</details>

Just chiming in, I can reproduce this aswel, using 5G mobile connection (so no router involved except for my iphone):
```sh
nix build github:nixos/nixpkgs/release-24.05/#python311Packages.dnspython
warning: Ignoring setting 'auto-allocate-uids' because experimental feature 'auto-allocate-uids' is not enabled
warning: Ignoring setting 'impure-env' because experimental feature 'configurable-impure-env' is not enabled
error: builder for '/nix/store/ar2ryb4ighb0x0wlzkq0yb5947xa8slz-python3.11-dnspython-2.6.1.drv' failed with exit code 1;
       last 10 log lines:
       > E       AssertionError: False is not true
       >
       > tests/test_resolver_override.py:104: AssertionError
       > ----------------------------- Captured stdout call -----------------------------
       > NOT EQUIVALENT
       > [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8888', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8844', 53, 0, 0))]
       > [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.8.8', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.4.4', 53, 0, 0))]
       > =========================== short test summary info ============================
       > FAILED tests/test_resolver_override.py::OverrideSystemResolverTestCase::test_basic_getaddrinfo - AssertionError: False is not true
       > ========== 1 failed, 1292 passed, 51 skipped, 1 deselected in 50.36s ===========
       For full logs, run 'nix-store -l /nix/store/ar2ryb4ighb0x0wlzkq0yb5947xa8slz-python3.11-dnspython-2.6.1.drv'.
```
 
```sh
nix-shell -p nix-info --run "nix-info -m"
```
 - system: `"aarch64-darwin"`
 - host os: `Darwin 23.6.0, macOS 14.7`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.18.8`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`

 
 ```sh
readlink /nix/var/nix/profiles/per-user/root/channels/nixpkgs
/nix/store/81d6lv96pws78iijmmsra5ql8wxgb1cb-nixpkgs/nixpkgs
```

```sh
cat /nix/store/81d6lv96pws78iijmmsra5ql8wxgb1cb-nixpkgs/nixpkgs/.git-revision
09ec6a0881e1a36c29d67497693a67a16f4da573
```

`nix build github:nixos/nixpkgs/nixos-unstable/#python311Packages.dnspython --rebuild -L`

Is producing the same error as before:

<details>

```
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/7bsil10gd6zl6vy3fb87dns7aixncda3-dnspython-2.7.0.tar.gz
source root is dnspython-2.7.0
setting SOURCE_DATE_EPOCH to timestamp 1580601600 of file dnspython-2.7.0/util/generate-rdatatype-doc.py
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
[1m* Getting build dependencies for wheel...[0m
[1m* Building wheel...[0m
[1m[92mSuccessfully built [4mdnspython-2.7.0-py3-none-any.whl[0m[1m[92m[0m
Finished creating a wheel...
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for dnspython-2.7.0-py3-none-any.whl
Finished executing pythonRuntimeDepsCheck
Running phase: installPhase
@nix { "action": "setPhase", "phase": "installPhase" }
Executing pypaInstallPhase
Successfully installed dnspython-2.7.0-py3-none-any.whl
Finished executing pypaInstallPhase
Running phase: pythonOutputDistPhase
@nix { "action": "setPhase", "phase": "pythonOutputDistPhase" }
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
Running phase: fixupPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
checking for references to /private/tmp/nix-build-python3.11-dnspython-2.7.0.drv-0/ in /nix/store/njkg929clr0kbp63k4dir9vx1pzm6w7h-python3.11-dnspython-2.7.0...
patching script interpreter paths in /nix/store/njkg929clr0kbp63k4dir9vx1pzm6w7h-python3.11-dnspython-2.7.0
stripping (with command strip and flags -S) in  /nix/store/njkg929clr0kbp63k4dir9vx1pzm6w7h-python3.11-dnspython-2.7.0/lib
checking for references to /private/tmp/nix-build-python3.11-dnspython-2.7.0.drv-0/ in /nix/store/s27si5shps9qmj6vhz84w92vr86ck5vx-python3.11-dnspython-2.7.0-dist...
patching script interpreter paths in /nix/store/s27si5shps9qmj6vhz84w92vr86ck5vx-python3.11-dnspython-2.7.0-dist
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
@nix { "action": "setPhase", "phase": "installCheckPhase" }
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
Running phase: pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
Running phase: pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: dns
Running phase: pytestCheckPhase
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Executing pytestCheckPhase
[1m============================= test session starts ==============================[0m
platform darwin -- Python 3.11.10, pytest-8.3.3, pluggy-1.5.0
rootdir: /private/tmp/nix-build-python3.11-dnspython-2.7.0.drv-0/dnspython-2.7.0
configfile: pyproject.toml
[1mcollecting ... [0m[1m
collecting 6 items                                                             [0m[1m
collecting 98 items                                                            [0m[1m
collecting 1115 items                                                          [0m[1m
collected 1326 items / 1 deselected / 1325 selected                            [0m

tests/test_address.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                               [  0%][0m
tests/test_async.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[33ms[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m   [  4%][0m
tests/test_bugs.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                               [  4%][0m
tests/test_constants.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                            [  4%][0m
tests/test_ddr.py [32m.[0m[32m.[0m[32m                                                     [  5%][0m
tests/test_dnssec.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m [  8%]
[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                                         [ 10%][0m
tests/test_dnssecalgs.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                       [ 10%][0m
tests/test_doh.py [33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[32m                                              [ 11%][0m
tests/test_doq.py [33ms[0m[33ms[0m[32m                                                     [ 11%][0m
tests/test_edns.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                     [ 12%][0m
tests/test_entropy.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                              [ 13%][0m
tests/test_exceptions.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                           [ 13%][0m
tests/test_features.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                              [ 14%][0m
tests/test_flags.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                         [ 14%][0m
tests/test_generate.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                 [ 16%][0m
tests/test_grange.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                          [ 16%][0m
tests/test_immutable.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                           [ 17%][0m
tests/test_message.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m [ 21%]
[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                          [ 23%][0m
tests/test_name.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m [ 27%]
[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m [ 32%]
[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[33ms[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                       [ 36%][0m
tests/test_namedict.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                            [ 38%][0m
tests/test_nsec3.py [32m.[0m[32m.[0m[32m                                                   [ 38%][0m
tests/test_nsec3_hash.py [32m.[0m[32m.[0m[32m.[0m[32m                                             [ 38%][0m
tests/test_ntoaaton.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m  [ 42%][0m
tests/test_processing_order.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                [ 43%][0m
tests/test_query.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m  [ 47%][0m
tests/test_rdata.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m [ 50%]
[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                [ 54%][0m
tests/test_rdataset.py [32m.[0m[32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                [ 55%][0m
tests/test_rdtypeandclass.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                  [ 57%][0m
tests/test_rdtypeanydnskey.py [32m.[0m[32m.[0m[32m                                         [ 57%][0m
tests/test_rdtypeanyeui.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                      [ 59%][0m
tests/test_rdtypeanyloc.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                          [ 59%][0m
tests/test_rdtypeanytkey.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                         [ 59%][0m
tests/test_renderer.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                                             [ 60%][0m
tests/test_resolution.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m                 [ 62%][0m
tests/test_resolver.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m [ 66%]
[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[32m                                                            [ 67%][0m
tests/test_resolver_override.py [31mF[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                           [ 68%][0m
tests/test_rrset.py [32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                               [ 70%][0m
tests/test_rrset_reader.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                                  [ 70%][0m
tests/test_serial.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                                         [ 71%][0m
tests/test_set.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m    [ 75%][0m
tests/test_svcb.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                                      [ 76%][0m
tests/test_tokenizer.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m [ 80%]
[0m[31m                                                                         [ 80%][0m
tests/test_transaction.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m [ 83%]
[0m[32m.[0m[32m.[0m[31m                                                                       [ 84%][0m
tests/test_tsig.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                                [ 85%][0m
tests/test_tsigkeyring.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                                        [ 86%][0m
tests/test_ttl.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                                              [ 86%][0m
tests/test_update.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                             [ 88%][0m
tests/test_wire.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                                               [ 89%][0m
tests/test_xfr.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[33ms[0m[33ms[0m[33ms[0m[31m                        [ 91%][0m
tests/test_zone.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33ms[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m [ 95%]
[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                         [ 99%][0m
tests/test_zonedigest.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[31m                                     [100%][0m

=================================== FAILURES ===================================
[31m[1m____________ OverrideSystemResolverTestCase.test_basic_getaddrinfo _____________[0m

self = <tests.test_resolver_override.OverrideSystemResolverTestCase testMethod=test_basic_getaddrinfo>

    @unittest.skipIf(
        sys.platform == "win32", "avoid windows original getaddrinfo issues"
    )
    def test_basic_getaddrinfo(self):
        self.assertTrue(
            self.equivalent("dns.google", 53, socket.AF_INET, socket.SOCK_DGRAM)
        )
>       self.assertTrue(
            self.equivalent("dns.google", 53, socket.AF_INET6, socket.SOCK_DGRAM)
        )
[1m[31mE       AssertionError: False is not true[0m

[1m[31mtests/test_resolver_override.py[0m:104: AssertionError
----------------------------- Captured stdout call -----------------------------
NOT EQUIVALENT
[(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8888', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('2001:4860:4860::8844', 53, 0, 0))]
[(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.8.8', 53, 0, 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_DGRAM: 2>, 17, '', ('::ffff:8.8.4.4', 53, 0, 0))]
[36m[1m=========================== short test summary info ============================[0m
[31mFAILED[0m tests/test_resolver_override.py::[1mOverrideSystemResolverTestCase::test_basic_getaddrinfo[0m - AssertionError: False is not true
[31m=========== [31m[1m1 failed[0m, [32m1271 passed[0m, [33m53 skipped[0m, [33m1 deselected[0m[31m in 6.86s[0m[31m ===========[0m

```

<deailts/>

--- a/pkgs/development/python-modules/dnspython/default.nix
+++ b/pkgs/development/python-modules/dnspython/default.nix
@@ -60,6 +60,12 @@ buildPythonPackage rec {
     "test_misc_good_WKS_text"
   ];
 
+  # disable network on all builds (including darwin)
+  # see https://github.com/NixOS/nixpkgs/issues/356803
+  preCheck = ''
+    export NO_INTERNET=1
+  '';
+
   pythonImportsCheck = [ "dns" ];
 
   meta = with lib; {

